### PR TITLE
[GOV- 267] Changing custom data type to array

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- Changing custom data type to array
Jira Ticket : https://mifosforge.jira.com/browse/GOV-267

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj @avikganguly01

**Review by fynmanoj — CHANGES_REQUESTED** [review]
how is this related to https://mifosforge.jira.com/browse/GOV-267

**Comment by truthfool** [comment]
> how does that connected to https://mifosforge.jira.com/browse/GOV-267

The array of custom data gets published.

**Review by fynmanoj — APPROVED** [review]

File: src/main/java/org/mifos/connector/common/channel/dto/TransactionChannelRequestDTO.java [hunk 1]
```diff
@@ -35,13 +35,13 @@ public class TransactionChannelRequestDTO {
     private String note;
     private String expiration;
     private ExtensionList extensionList;
-    private CustomData customData;
+    private List<CustomData> customData;
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/common/channel/dto/TransactionChannelRequestDTO.java:38]: the naming convention is wrong

**Pull request merged**